crates/sui-types/src/object.rs

```diff
@@ -482,6 +482,32 @@ pub enum Owner {
     },
     /// Object is immutable, and hence ownership doesn't matter.
     Immutable,
+    /// Object is sequenced via consensus. Ownership is managed by the configured authenticator.
+    ///
+    /// Note: wondering what happened to `V1`? `Shared` above was the V1 of consensus objects.
+    ConsensusV2 {
+        /// The version at which the object most recently became a consensus object
```

Comment:
Worth saying more explicitly that this performs the identical function as `initial_shared_version`